### PR TITLE
Update README.MD

### PR DESCRIPTION
**Pull request by rkSlalom**
Added instructions to create database and corrected Step 3 to run the correct project file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,14 @@ Install the latest verison of .NET Core for Windows/Linux or Mac.
 
 1. Install SQL Server Developer 2019
     > https://www.microsoft.com/en-us/sql-server/sql-server-downloads
-2. Update connection string server name and credentials in [appsettings.json](./src/BlackSlope.Api/appsettings.json)
+2. Run Create DATABASE movies command to create the database locally .Update connection string server name and
```
Comment (pmotameni) [inline README.md]: You do not need to create the database first, ef migration process should create it.

Comment (sgriegoliet) [reply README.md]: I'm not too familiar with EF Migration, but I had to manually create my Movies database for the db-update script to succeed.

Comment (pmotameni) [reply README.md]: It should not be the case. What is your DB server?

Comment (sgriegoliet) [reply README.md]: A localdb on SQL Server 2019

Comment (theczorn) [reply README.md:24]: This work has been patched in prior PR's.

[See here.](https://github.com/SlalomBuild/blackslope.net/pull/46/files)

**Comment by theczorn** [comment]
https://github.com/SlalomBuild/blackslope.net/pull/46/files

^ ReadMe updated in prior PR. Closing as duplicate

**Pull request closed without merging**